Well -- the regular -- codespell support

yarikoptic opened the pull request:
may be no need for workflow since pre-commit used?

yarikoptic commented:
should I just redo it clean with the "state of the art"? ;)

yarikoptic committed:
Add github action to codespell dev on push and PRs

yarikoptic committed:
Add rudimentary codespell config

yarikoptic committed:
Add pre-commit definition for codespell

yarikoptic commented:
faster done than said -- force-pushed

effigies commented on docs/examples/models.rst:
```diff
@@ -42,11 +42,11 @@ Dataset: https://openneuro.org/datasets/ds000030/versions/00016
 
 In this model, we contrast responses to ``ACCEPT`` and ``EXPLODE`` actions
-in ``BALOON`` trials only.
+in ``BALLOON`` trials only.
```
We should exclude this one, as it reflects the contents of the dataset. Hence the "\[*sic*\]".

effigies commented on examples/models/ds000030/models/model-001_smdl.json:
```diff
@@ -28,7 +28,7 @@
                     "Name": "And",
                     "Input": [
-                        "trial_type.BALOON",
+                        "trial_type.BALLOON",
```
And here.

yarikoptic replied:
done done

yarikoptic committed:
ignore baloon

yarikoptic committed:
[DATALAD RUNCMD] chore: run codespell throughout fixing a few  new typos automagically

=== Do not change lines below ===
{
 "chain": [],
 "cmd": "codespell -w",
 "exit": 0,
 "extra_inputs": [],
 "inputs": [],
 "outputs": [],
 "pwd": "."
}
^^^ Do not change lines above ^^^
